thrift/worldinterface/worldinterface_rpc.thrift

```diff
@@ -153,13 +153,31 @@ service WorldInterfaceServer {
     bool attach(1:string id, 2:string link_name);
     
     /**
+    * Attach an object to a link of the robot - takes unscoped names as arguments
+    * @param object_name string that identifies object in gazebo (returned after creation or spawnning)
```

Comment:
I double checked again, these methods can be used to attach/detach any objects spawned in gazebo world, not just limited to objects created using world interface